Makefile

```diff
@@ -2,6 +2,7 @@ SHELL := /bin/bash
 export GO111MODULE ?= on
 export VERSION ?= v1.6.0
 export ECE_VERSION ?= $(shell cat ECE_VERSION)
+export ECE_BRANCH ?= 62
```

Comment:
Why not keeping the `ms-` prefix? Looks like you are adding back the `ms-` prefix in `scripts/update-swagger.sh`